src/TriggerBinding/SqlTriggerTargetScaler.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql
+{
+    public class SqlTriggerTargetScaler : ITargetScaler
```

Comment:
Add doc comment explaining what this class is used for (ideally with links to public docs about scaling)

Reply:
Added doc comment. As target based scaling has not yet rolled out, couldn't get any public docs for it.